scripts/clear-npm-script.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+const fs = require('fs');
```

Comment:
I can see what this is doing based on the pr-build.yml file, but it might be helpful to add a comment at the top of this file explaining why it exists

Reply:
Okay, I'll add a comment.

Reply:
Thank you - this will be helpful for those of us who aren't you or Steve if we have to wade into the build code :)

Reply:
I added a comment. Let me know what you think.

Reply:
LGTM, TY